Backend/config/postgres.js

```diff
@@ -27,8 +27,8 @@ loadModels(db, sequelize);
 
 dbConnectionTest().then(async () => {
   if (
-    process.env.DB_SYNC_FORCE == "true" &&
-    process.env.NODE_ENV !== "production"
+    process.env.DB_SYNC_FORCE == "true"
+    // && process.env.NODE_ENV === "development"
```

Comment:
Hier muss ich mir noch was einfallen lassen, wie man die DB in production korrekt initalisiert.

Du musst z.Zt. sicher gehen, dass du die Option FORCE=true wirklich nur beim ersten Hochfahren gesetzt ist. Da die Container ja auf restart stehen, würde sonst die DB jedes mal gelöscht werden, wenn der Server crashed.

Andereseits könnten wir uns auch überlegen, ob wir aktuell eh nicht mit einem Cron-Job jeden Tag um 00:00 die Umgebung neustarten. Dann würden aúch die Testdaten neu eingespielt und die Daten der Flüge für die Tageswertung wieder angepasst.

Alternativ, könnte man den TestDataController in app.js aus dem If nehmen. Dann könnte man api/testdata/seed nutzen.